packages/material-ui-types/package.json

```
@@ -33,5 +33,8 @@
   "sideEffects": false,
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "@types/react": "^16.8.20"
```

```suggestion
    "@types/react": "*"
```

Type dependencies are hacks currently. In a perfect world those would be peers. This follows `@types/*` best practices.